mmr-backend/test/sample-data/books.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "id": "ABookId",
```

Comment:
I'll make it out of curiosity but since this is just for tests purposes, representing the output of the layer that will transform the received data from a source to a format my code understands, I don't consider it necessary at this point

Reply:
I'm good as far as it's a MVP, but you should at least check out the format and fields returned by some API. They might be somewhat different, in type or other kind.